spec/marketing.json

```diff
@@ -1982,7 +1983,7 @@
                         "name": "before_create_time",
                         "x-title": "Before Create Time",
                         "in": "query",
-                        "description": "Restrict the response to automations created before the set time. We recommend [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) time format: 2015-10-21T15:41:36+00:00.",
+                        "description": "Restrict the response to automations created before this time. Uses the ISO 8601 time format: 2015-10-21T15:41:36+00:00.",
```

Comment:
Should this (and the other mentions of ISO 8601 be updated to reflect the date-time convention fixes we made below?

Reply:
Yeah possibly. I'll push this out, but check on that